src/LuaTransformer.ts

```diff
@@ -1267,11 +1520,55 @@ export class LuaTransformer {
     }
 
     public transformPostfixUnaryExpression(expression: ts.PostfixUnaryExpression): tstl.Expression {
-        throw new Error("Method not implemented.");
+        return this.transformCompoundAssignmentExpression(
+            expression.operand,
+            ts.createLiteral(1),
+            expression.operator === ts.SyntaxKind.PlusPlusToken
```

Comment:
Ternaries inside arguments are to very readable imo. Consider changing it to:

```
        let operator = expression.operator === ts.SyntaxKind.PlusPlusToken ? tstl.SyntaxKind.AdditionOperator : tstl.SyntaxKind.SubractionOperator;
        return this.transformCompoundAssignmentExpression(
            expression.operand,
            ts.createLiteral(1),
            operator,
            true
````